### ISSUE TITLE
Successfully train up the yolo on my custom data

### ISSUE THREAD

**Issue by jaysonph** [eriklindernoren/PyTorch-YOLOv3#398]
Recently, Successfully train up the yolo on my custom data. I have added some of my own augmentations and added some useful training callbacks (modelcheckpoint, LR scheduler, earlystopping). The training result is quite accurate and comparable to the previous results I got from keras yolov3.

Knowing author might not have time to answer all the problems, the code is very good and well-structured. See if many people here need it, I might take one day to upload it.
